extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -30,9 +30,9 @@ import {isEnumValue} from '../../../src/types';
 import {Activity} from './activity-impl';
 import {Cid} from './cid-impl';
 import {
-    InstrumentationService,
-    instrumentationServicePromiseForDoc,
-    AnalyticsEventType,
+  InstrumentationService,
```

Comment:
Yeah, I guess this is my ide. Lint accepts either way. I rolled back unrelated indentation changes.